src/apis/modTools.es6.js

```diff
@@ -1,19 +1,34 @@
 import apiRequest from '../apiBase/apiRequest';
 
+const DISTINGUISH_TYPE = {
+  'moderator': 'yes',
+  '': 'no',
+  'admin': 'admin',
+};
+
```

Comment:
I think ideally we'd export `DISTINGUISH_TYPE` here as it's intended to be used in the client app, e.g.

```javascript
/**
  * @enum
  */
export DISTINGUISH_TYPE = {
  NONE: '',
  MODERATOR: 'moderator',
  ADMIN: 'admin',
};
```

and import that rather than redefining it in the client app.  Then, handle the mapping of `DISTINGUISH_TYPE.MODERATOR → 'yes'` and `DISTINGUISH_TYPE.NONE → 'no'` for the API call within the `distinguish` function.